deploy/manifests/dev/us-east-2/cluster/storetheindex/github-auth.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: Secret
+metadata:
+    name: github-auth
+type: Opaque
+stringData:
+    username: ENC[AES256_GCM,data:smZYA3XROQ==,iv:V6zGnaSjbGixZlkYiQiDReK8PQLYjC4fO8V9WzXvp3o=,tag:H+UZsHgVnvS1GDc1IjIuQQ==,type:str]
```

Comment:
this file looks painful to regenerate

Reply:
I am curious; why is it painful?

Reply:
maybe i just don't know the tool you used to encrypt these into this format, but i don't think i would know how to replicate it if i needed to change the password because
1. i don't know what the tool is
2. i don't know what key this data is encrypted under

Reply:
OK I think you probably haven't seen this: https://github.com/filecoin-project/storetheindex/tree/main/deploy/manifests